std/assembly/math.ts

```diff
@@ -2128,16 +2115,14 @@ export namespace NativeMathf {
       n -= 127;
       if (n > 127) {
         y *= Ox1p127f;
-        n -= 127;
-        if (n > 127) n = 127;
+        n = builtin_min<i32>(n - 127, 127);
       }
     } else if (n < -126) {
-      y *= Ox1p_126f;
-      n += 126;
+      y *= Ox1p_126f * Ox1p24f;
+      n += 126 - 24;
```

Comment:
What's this doing?

Reply:
See original code:
https://git.musl-libc.org/cgit/musl/tree/src/math/scalbnf.c

Reply:
It seems this fix appear later in original source code. Ported "musl" don't reflect this changes

Reply:
Oh, updates, nice :)